src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+import {LocalizedLabels, LocalizedSteps, registerLocalization} from '../localization';
+
+const uiLabels: LocalizedLabels = {
+  'Angular Update Guide': 'Hướng dẫn Update Angular',
+  'Select the options matching your project:': 'Tuỳ chỉnh những lựa chọn cho phù hợp với dự án của bạn:',
+  'Angular Versions': 'Phiên bản Angular',
+  From: 'Từ',
+  To: 'Đến',
+  Warning: 'Chú ý',
+  'Plans for releases after the current major release are not finalized and may change. These recommendations are based on scheduled deprecations.':
+    'Đối với những phiên bản Angular được phát hành sau phiên bản chính thức hiện tại, kế hoạch và dự định cho những phiên bản này chưa được thông qua chính thức và có thể sẽ thay đổi. Những gợi ý sau dựa trên những chỉnh sửa đã được thống nhất.',
+  'We do not currently support downgrading versions of Angular.':
+    'Chúng tôi hiện tại không hỗ trợ Downgrade phiên bản Angular',
+  'We do not recommend moving across multiple major versions.':
+    'Chúng tôi không khuyến khích update Angular theo kiểu "nhảy cóc" phiên bản.',
+  'App Complexity': 'Độ phức tạp của ứng dụng',
+  Basic: 'Cơ bản',
+  Medium: 'Vừa',
+  Advanced: 'Nâng cao',
+  'Show update information relevant to all Angular developers.':
+    'Hiển thị thông tin update dành cho tất cả mọi tầng lớp lập trình viên Angular.',
+  'Show more specific update information relevant to around 10% of Angular developers.':
+    'Hiển thị thông tin update chi tiết hơn dành cho khoảng 10% lập trình viên Angular. ',
+  'Show all the information we have about the update.': 'Hiển thị tất cả thông tin về việc update mà chúng tôi biết được.',
+  'Other Dependencies': 'Những yêu cầu liên quan khác',
+  'I use': 'Tôi sử dụng',
+  'Package Manager': 'Package Manager',
```

Comment:
Package Manager = "Phần mềm quản lý thư viện"

Reply:
I would say we should keep as Package Manager :)

Reply:
I'd agree with keeping it as "Package Manager". I'll put a comment with a list of terms I personally think we should keep as English because the point is to help people understand the guide. Some terms aren't translatable or if translated, will take a load of mental energy to understand.